executor/executor_test.go

```diff
@@ -165,49 +170,97 @@ func dummyCommandSpec(format string) (spec *util.CommandSpec) {
 }
 
 func TestNew(t *testing.T) {
-	// case binary format
-	spec := dummyCommandSpec(binaryFormat)
-	sdapi := newDummySDAPI(spec, nil)
-	// success
-	executor, err := New(sdapi, []string{"ns/cmd@ver"})
-	if err != nil {
-		t.Errorf("err=%q, want nil", err)
+	successCases := []struct {
```

Comment:
Do we need to add a test case that check  the file is not empty, config.Debug = true.

Reply:
Thx. I added it.